### PR TITLE
KEYCLOAK-6796 Not able to run tests without -Pwildfly-managed

### PR DESCRIPTION
**Pull request by martin-kanis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,6 +58,7 @@
         <arquillian-managed>true</arquillian-managed>
         <jboss-cli.executable>./jboss-cli.sh</jboss-cli.executable>
         <keycloak.management.port>10090</keycloak.management.port>
+        <skipTests>true</skipTests>
```
Comment (abstractj) [inline pom.xml]: @martin-kanis I'm not sure if it's a good idea to skipTests by default for any project. What about create a separate profile for Spring? One idea you can try to have -Pwildfly-managed could be the default profile if nothing is specified, and if -Pspring-boot is specified the default profile is deactivated.

I understand what was your idea here, but I think would be better to try separate profiles.

File: app-authz-rest-springboot/README.md [hunk 2]
```diff
@@ -169,4 +169,4 @@ Integration test of the Quickstart
 1. Make sure you have an Keycloak server running with an admin user in the `master` realm or use the provided docker image
 2. Be sure to set the `TestHelper.keycloakBaseUrl` in the `createArchive` method (default URL is localhost:8180/auth).
 3. Set accordingly the correct url for the `keycloak.auth-server-url` in the test [application.properties](src/test/resources/application.properties).
-4. Run `mvn test`
\ No newline at end of file
+4. Run `mvn test -DskipTests=false`
```
Comment (abstractj) [inline app-authz-rest-springboot/README.md]: I find the need to specify skip tests to false is confusing and a bit redundant. If you're providing `mvn test` it is implicit that we don't want to skip these tests.

File: service-springboot-rest/README.md [hunk 3]
```diff
@@ -100,5 +100,5 @@ Integration test of the Quickstart
 1. Make sure you have an <span>Keycloak</span> server running with an admin user in the `master` realm or use the provided docker image
 2. Be sure to set the `TestHelper.keycloakBaseUrl` in the `createArchive` method (default URL is localhost:8180/auth).
 3. Set accordingly the correct url for the `keycloak.auth-server-url` in the test [application.properties](src/test/resources/application.properties).
-4. Run `mvn test`
+4. Run `mvn test -Pspring-boot`
```
Comment (abstractj) [inline service-springboot-rest/README.md]: @martin-kanis did you get the following output while running tests?
```
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.43 sec <<< FAILURE! - in org.keycloak.quickstart.springboot.ProductServiceTest
org.keycloak.quickstart.springboot.ProductServiceTest  Time elapsed: 1.429 sec  <<< ERROR!
javax.ws.rs.BadRequestException: HTTP 400 Bad Request
        at org.keycloak.quickstart.springboot.ProductServiceTest.setup(ProductServiceTest.java:32)

org.keycloak.quickstart.springboot.ProductServiceTest  Time elapsed: 1.429 sec  <<< ERROR!
javax.ws.rs.NotFoundException: HTTP 404 Not Found
        at org.keycloak.quickstart.springboot.ProductServiceTest.cleanUp(ProductServiceTest.java:60)


Results :

Tests in error:
org.keycloak.quickstart.springboot.ProductServiceTest.org.keycloak.quickstart.springboot.ProductServiceTest
  Run 1: ProductServiceTest.setup:32 » BadRequest HTTP 400 Bad Request
  Run 2: ProductServiceTest.cleanUp:60 » NotFound HTTP 404 Not Found
```

Comment (martin-kanis) [reply service-springboot-rest/README.md]: I've run the tests for each quickstart related to this PR separately and everything passed. How did you run the tests?

Comment (sebastienblanc) [reply service-springboot-rest/README.md]: It's running fine for me. @abstractj Stupid question but was your KC server running ?

**Comment by abstractj** [comment]
@martin-kanis I also suggest you to change `travis-run-tests.sh` to: 

```
if [ $1 == "group4" ]; then
  cd app-authz-springboot && mvn -B -s ../maven-settings.xml clean test -Pspring-boot
  cd app-authz-rest-springboot && mvn -B -s ../maven-settings.xml clean test -Pspring-boot
  cd ../service-springboot-rest && mvn -B -s ../maven-settings.xml clean test -Pspring-boot
  mvn spring-boot:run&
  cd ../app-springboot
  mvn -B -s ../maven-settings.xml clean test -Pspring-boot
fi

```

**Comment by martin-kanis** [comment]
@abstractj added

File: service-springboot-rest/README.md (hunk 3 above)
Comment (abstractj) [reply service-springboot-rest/README.md]: @sebastienblanc yes! Let's just wait for the output from Travis. I believe you and Travis are more reliable than me when it comes to Spring Boot.

**Pull request merged**